src/components/Form.js

```diff
@@ -85,44 +76,40 @@ const StyledLabel = styled.label`
   display: flex;
   flex-direction: column;
   flex-wrap: wrap;
-  margin-bottom: 20px;
   width: 100%;
+  margin-top: 20px;
 
   ${minBreakpointQuery.smedium`
-    &:nth-child(1) {
-      grid-area: 1 / 1 / 2 / 2;
-    }
-    &:nth-child(2) {
-      grid-area: 1 / 2 / 2 / 3;
-    }
     &:nth-child(3) {
-      grid-area: 2 / 1 / 3 / 3;
+      grid-column: span 2
```

Comment:
This is all better, but a prop might be even more useful just so it's more explicit, take a look at StyledTab on https://github.com/attachdigital/base-gatsby-website/blob/master/src/components/Tabs.js